tools/pot/.gitignore

```diff
@@ -105,3 +105,6 @@ ENV/
 
 # snapshots
 *.tar
+
+# Autogenerated version file
```

Comment:
I think it is a valuable change, but it shouldn't be part of this PR.

Reply:
removed